integrationtests/system_test.go

```diff
@@ -67,36 +64,34 @@ func TestServiceCommands(t *testing.T) {
 		require.NoError(t, err, "failed unmarshalling json out=%v", out)
 
 		assert.Equal(t, serviceInfo.Status, uint32(response.Status))
-		assert.Equal(t, v1alpha1.ServiceStatus_STOPPED, response.Status)
+		assert.Equal(t, system.SERVICE_STATUS_STOPPED, response.Status)
```

Comment:
Nice, I see that this was generated from protoc but replaced with a const that was already defined in pkg/system/types.go